src/Component/Authentication/Forgot.jsx

```diff
@@ -0,0 +1,264 @@
+import React, { useState } from "react";
+import Alert from "@mui/material/Alert";
+import TextField from "@mui/material/TextField";
+import Button from "@mui/material/Button";
+import LoginIcon from "@mui/icons-material/Login";
+import {
+  Container,
+  Dialog,
+  DialogContent,
+  DialogTitle,
+  Typography,
+  SvgIcon,
+} from "@mui/material";
+import Box from "@mui/material/Box";
+import CloseIcon from "@mui/icons-material/Close";
+import IconButton from "@mui/material/IconButton";
+import Paper from "@mui/material/Paper";
+import Chip from "@mui/material/Chip";
+import Link from "@mui/material/Link";
+import { TypeSpecimenOutlined } from "@mui/icons-material";
+
+const isEmail = (email) =>
+  /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(email);
+
+function Forgot() {
+  const [emailInput, setEmailInput] = useState();
+
+  const [emailError, setEmailError] = useState(false);
+
+  const [formValid, setFormValid] = useState();
+  const [success, setSuccess] = useState();
+
+  const [open, setOpen] = useState(true);
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const handleLoginEmail = () => {
+    if (!isEmail(emailInput)) {
+      setEmailError(true);
+      return;
+    }
+    setEmailError(false);
+  };
+
+  const handleLoginSubmit = (e) => {
+    e.preventDefault();
+    setSuccess();
+
+    if (emailError || !emailInput) {
+      setFormValid("Email is inValid.Please Re-Enter Email");
+      return;
+    }
+
+    setFormValid(null);
+    setSuccess("Form submitted successfully");
+
+    console.log("Email:" + emailInput);
+  };
+
+  return (
+    <>
+      <Box
+        component="form"
+        sx={{
+          height: "100vh",
+          display: "flex",
+          flexWrap: "wrap",
+          justifyContent: "center",
+          alignItems: "center",
+        }}
+      >
+        <Dialog
+          open={open}
+          onClose={handleClose}
```

Comment:
Would it be possible to update onClose={null} so that the popup only closes when the close button is clicked, rather than when clicking outside of it? For all pop-ups

Reply:
As per comment I changed to onClose={null} and it works.Thanks @nihalerdal